backend/src/core/config.ts

```diff
@@ -300,11 +300,18 @@ const config = convict({
     format: 'int',
   },
   transactionalEmail: {
-    rate: {
+    userRate: {
+      doc:
+        'The max number of transactional emails that can be requested per window per user',
+      default: 5,
+      env: 'TRANSACTIONAL_EMAIL_USER_RATE',
+      format: 'int',
+    },
+    globalRate: {
       doc:
         'The max number of transactional emails that can be requested per window',
       default: 10,
```

Comment:
I think we can definitely bump this up to higher. We can send 120m/s. Most of the time the capacity is not utilised.

Reply:
I've set this to 10, but I guess we'll revise this after talking to users.

Reply:
I've increased this to 10, but I guess we'll increase this after understanding users' usage patterns.